reference/tax_provider.yml

```diff
@@ -659,11 +659,14 @@ components:
           $ref: '#/components/schemas/TaxClass'
         tax_exempt:
           type: boolean
-          description: Flag whether or not this item is always tax exempt. Gift Certificate purchases for example. Tax Exempt items should be included in the Request for auditing purposes.
+          description: Flag whether or not this item is always tax exempt. For example, for gift certificate purchases and order-level refunds. Tax exempt items are included in the request for auditing purposes.
```

Comment:
🍹 For example, gift certificate purchases and order-level refunds are tax exempt.

Reply:
Thanks for the suggestion, updated in this [commit](https://github.com/bigcommerce/api-specs/pull/309/commits/f1494fc63c72e5c63dd648d7fb7e30f02fc930ad).